src/components/pages/import-batch-view.js

```diff
@@ -78,10 +78,10 @@ function ImportBatchView({ initialState, PageTemplate }) {
   };
 
   const renderDate = date => <span>{formatDate(date)}</span>;
-  const renderTitle = ({ title, slug }) => (
+  const renderTitle = ({ title, key, slug }) => (
     <a
       target="_blank"
-      href={getImportedArticleUrl({ hostName, allowUnsecure, slug })}
+      href={getImportedArticleUrl({ hostName, allowUnsecure, key, slug })}
```

Comment:
I guess this becomes `getImportedDocUrl` since we're removing the concept of article, right?

Reply:
Fixed

Reply:
Fixed